From 214bc348d60a389cb7d18cbc2602a64301c8932a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Mar 2018 15:37:25 +0800
Subject: [PATCH 3448/5242] MLK-17698-2 ARM64: dts: freescale: imx8qxp: update
 thermal zone info

commit  bad6509759769dedae79ada79414b710152b39f2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update thermal zone number, including CPU thermal
zone and DRC thermal zone.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |   22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index c3a3374..1c294e2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright 2017-2018 NXP
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -880,7 +880,7 @@
 	tsens: thermal-sensor {
 		compatible = "nxp,imx8qxp-sc-tsens";
 		/* number of the temp sensor on the chip */
-		tsens-num = <1>;
+		tsens-num = <2>;
 		#thermal-sensor-cells = <1>;
 	};
 
@@ -911,6 +911,24 @@
 				};
 			};
 		};
+
+		drc-thermal0 {
+			polling-delay-passive = <250>;
+			polling-delay = <2000>;
+			thermal-sensors = <&tsens 1>;
+			trips {
+				drc_alert0: trip0 {
+					temperature = <85000>;
+					hysteresis = <2000>;
+					type = "passive";
+				};
+				drc_crit0: trip1 {
+					temperature = <127000>;
+					hysteresis = <2000>;
+					type = "critical";
+				};
+			};
+		};
 	};
 
 	irqsteer_csi: irqsteer@58220000 {
-- 
1.7.9.5

